spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/PubSubAdmin.java

```diff
@@ -196,8 +200,14 @@ public Subscription createSubscription(String subscriptionName, String topicName
 
 		int finalAckDeadline = this.defaultAckDeadline;
 		if (ackDeadline != null) {
-			Assert.isTrue(ackDeadline >= 0,
-					"The acknowledgement deadline value can't be negative.");
+			Assert.isTrue(ackDeadline >= MIN_ACK_DEADLINE_SECONDS
```

Comment:
probably makes sense to move that check to a separate method this to avoid duplication